### ISSUE TITLE
VarNet paper knee SSIM results

### ISSUE THREAD

**Issue by z-fabian** [facebookresearch/fastMRI#74]
I am trying to reproduce results for the multicoil knee data using VarNet. I am a bit confused by the discrepancy between the SSIM reported in Table 2 for 8x acceleration with random mask (0.878) and the leaderboard SSIM for the same acceleration in Table 3 (0.890). 

Based on the paper's description and the code base I found the following differences between the two experiments:
- Table 2 result is trained on training data only, Table 3 is trained on training + val data
- Table 2 result is trained for 50 epochs, Table 3 result for 100 epochs 
- it is not completely clear from the paper, but I assume Table 2 is reported on the validation set and Table 3 on the test set

My understanding is that knee test data also uses random masks and not equispaced.
I don't think these differences only may completely account for the discrepancy. Is there anything else I missed?

**Comment by anuroopsriram** [comment]
The three differences you mentioned explain the difference in the numbers. Including validation data during training improves results significantly.
